cmd/machines.go

```diff
@@ -58,6 +58,19 @@ func init() {
 				),
 			},
 			{
+				Name:      "destroy",
+				Usage:     "Destroy a machine in the specified organization",
+				ArgsUsage: "<identity>",
```

Comment:
we should specify the identity as a machine id or machine name (not sure how its done with `machines view` but `<identity>` doesn't really tell me "what an identity is"